spec/api/v1/schemas/profiles.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Schemas
+      module Profiles
```

Comment:
Most of these modules only define the resource schema, such as `PROFILE` here, but we can also create schemas for profile-related things, such as `PROFILE_RELATIONSHIPS` in the future. Right now, they're just defined inline in the `profile_spec.rb` test.